src/javascript/app/pages/user/account/settings/self_exclusion.js

```diff
@@ -294,6 +314,7 @@ const SelfExclusion = (() => {
     );
 
     const setExclusionResponse = (response) => {
+        const exclude_until_val = $exclude_until.attr('data-value');
```

Comment:
this should be declared above its usage so if response has an error we don't declare this for no reason